Sources/Just/Just.swift

```diff
@@ -1014,7 +1014,9 @@ public final class HTTP: NSObject, URLSessionDelegate, JustAdaptor {
     if isSynchronous {
       let timeout = timeout.flatMap { DispatchTime.now() + $0 }
         ?? DispatchTime.distantFuture
-      _ = semaphore.wait(timeout: timeout)
+      let timeIntervalBetweenThread: Double = 0.2
+      
+      _ = semaphore.wait(timeout: timeout + timeIntervalBetweenThread)
```

Comment:
Check the image,  assume that `timeout = 2`, and  `TaskConfiguration.completionHandler` is executed on thread **Ta**, `semaphore.wait(timeout` is executed on thread **Tb**, `semaphore.wait` and `request.resume` are executed at same time, after 2 seconds, they are timeout at the same time, but **Tb** will cost 0.001 second to handle the result, so , after 2.001 seconds, `semaphore.signal` is called. but `semaphore` is timeout at 2 seconds and return the result with the default error `syncResultAccessError`, so I add 0.2 seconds to make it have enough time to run `semaphore.signal`

![img_1494](https://user-images.githubusercontent.com/2714474/30301926-c935ef48-97b1-11e7-9862-cb963a2bf5e6.JPG)